ceno_rt/src/syscalls.rs

```diff
@@ -1,18 +1,18 @@
 // Based on https://github.com/succinctlabs/sp1/blob/013c24ea2fa15a0e7ed94f7d11a7ada4baa39ab9/crates/zkvm/entrypoint/src/syscalls/keccak_permute.rs
-
-const KECCAK_PERMUTE: u32 = 0x00_01_01_09;
-
+#[cfg(target_os = "zkvm")]
 use core::arch::asm;
 
+pub const KECCAK_PERMUTE: u32 = 0x00_01_01_09;
```

Comment:
I just moved this one down, because we typically put `use` declarations above almost everything else in our files.